README.md

```diff
@@ -1,2 +1,6 @@
 # JupyterLab-user-testing
 Repository containing infrastructure and testing scripts for JupyterLab user testing ✨
+
+## TLJH
+
```

Comment:
Need to add details such as versions, etc. and infra running.
From @costrouc `e2-standard-2 2cpu, 8GB ram`

Reply:
Added in latest commit.